README.md

```diff
@@ -277,6 +234,35 @@ browserify analytics/index.js --standalone rudderTyper >  rudderTyperBundle.js
 </script>
 ```
 
+#### 2. Using NPM
+
+Import the RudderTyper-generated client and make the calls if your framework supports them.
+
+```javascript
+// Import RudderStack JS SDK and initialize it
+import { RudderAnalytics } from '@rudderstack/analytics-js';
+// Import your auto-generated RudderTyper client:
+import { RudderTyperAnalytics } from './analytics/index';
+
+const rudderAnalytics = new RudderAnalytics();
+rudderAnalytics.load(WRITE_KEY, DATA_PLANE_URL, {});
+
+// Pass in your @rudderstack/analytics-js instance to RudderTyper client
+RudderTyperAnalytics.setRudderTyperOptions({
+  analytics: rudderAnalytics,
+});
+
+// Issue your first RudderTyper track call!
+RudderTyperAnalytics.orderCompleted({
+  orderID: 'ck-f306fe0e-cc21-445a-9caa-08245a9aa52c',
+  total: 39.99,
+});
+```
+
+> Note: Remember to replace all instances of `WRITE_KEY` and `DATA_PLANE_URL` in the snippet with your write key and data plane URL values respectively.
```

Comment:
Please move this to the CDN section.

Reply:
If they use the snippet from the RS dashboard, these values will be prefilled. So, no need for this instruction itself. Apologies if your original intention was the same.

If you think it'll be helpful for the NPM case, keep it but phrase it appropriately.

Reply:
Done